external/csmith/CMakeLists.txt

```diff
@@ -0,0 +1,25 @@
+# Copyright (c) Facebook, Inc. and its affiliates.
+#
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+
+project(gflags)
+cmake_minimum_required(VERSION 3.15)
+
+include(ExternalProject)
+
+# Note that in the Bazel build the commit hash is
```

Comment:
Same again here

Reply:
This was erroneous copy-paste comment. I removed it.